code/game/objects/items/crayons.dm

```diff
@@ -428,6 +441,46 @@
 	else
 		..()
 
+/obj/item/toy/crayon/proc/can_claim_for_gang(mob/user, atom/target)
+	var/area/A = get_area(target)
+	if(!A || (!is_station_level(A.z)))
+		to_chat(user, "<span class='warning'>[A] is unsuitable for tagging.</span>")
+		return FALSE
+
+	var/spraying_over = FALSE
+	for(var/obj/effect/decal/cleanable/crayon/gang/G in target)
+		spraying_over = TRUE
+
+	for(var/obj/machinery/power/apc in target)
+		to_chat(user, "<span class='warning'>You can't tag an APC.</span>")
+		return FALSE
+
+	var/occupying_gang = territory_claimed(A, user)
+	if(occupying_gang && !spraying_over)
+		to_chat(user, "<span class='danger'>[A] has already been tagged by a gang! You must find and spray over the old tag first!</span>")
```

Comment:
*You must find and spray over **or otherwise remove** the old tag first!

Reply:
```suggestion
		to_chat(user, "<span class='danger'>[A] has already been tagged by a gang! You must find and spray over or otherwise remove the old tag first!</span>")
```